CRP.Mvc/Controllers/HomeController.cs

```diff
@@ -12,6 +12,14 @@ namespace CRP.Controllers
     [HandleTransactionsManually]
     public class HomeController : ApplicationController
     {
+        public ActionResult IndexOld()
```

Comment:
I'll delete before live. Just want them (and me) to be able to compare with the old stuff.